client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -383,6 +383,7 @@ class CalypsoifyIframe extends Component<
 			const isExperimental = config.isEnabled( 'gutenboarding/feature-picker' ); // TODO: remove after ETK 2.8.6 is released
 			const isPersistentLaunchButton = config.isEnabled( 'create/persistent-launch-button' ); // TODO: remove after ETK 2.8.6 is released
 			const isFocusedLaunchFlow = config.isEnabled( 'create/focused-launch-flow' );
+			const currentCalypsoUrl = window.location.href;
```

Comment:
@razvanpapadopol is piggy backing on this to get the `currentCalypsoUrl` in the Focused Launch context OK or should we separate this out into a more explicit bit of code do you think?

Reply:
It seems like we're already transmitting this kind of data [a few lines below](https://github.com/Automattic/wp-calypso/blob/trunk/client/gutenberg/editor/calypsoify-iframe.tsx#L425-L430), information that is then used on the other side in [getCalypsoUrlInfo function](https://github.com/Automattic/wp-calypso/blob/trunk/apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js#L855).

It's shouldn't be a blocker for this PR if it works well. Just make sure it's documented well, at least in code, that we cut some corners.